### PR TITLE
Add blog post about viewing logs in a release app

### PR DESCRIPTION
**Pull request by bryanoltman**
# Status

**READY**

## Description

Adds a blog post describing how to view logs for a release app for both iOS and Android.

### CODE REVIEW

File: src/pages/blog/release-logs.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: Blog | Viewing logs in release apps
+description: How to view logs for an app without a debugger attached.
+date: August 20, 2024
+---
+
+# Viewing logs for your release app
```
Comment (felangel) [inline src/pages/blog/release-logs.md:8]:
```suggestion
# Viewing logs for a released app
```

File: src/pages/blog/release-logs.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: Blog | Viewing logs in release apps
```
Comment (felangel) [inline src/pages/blog/release-logs.md:3]:
```suggestion
title: Blog | Viewing logs in released apps
```

File: src/pages/blog/release-logs.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
```
Comment (felangel) [inline src/pages/blog/release-logs.md:1]: might be nice to rename this to `viewing-logs.md`

**Review by felangel — APPROVED** [review]

File: src/pages/blog/release-logs.md (hunk 3 above)
Comment (bryanoltman) [reply src/pages/blog/release-logs.md]: Done

File: src/pages/blog/release-logs.md (hunk 1 above)
Comment (bryanoltman) [reply src/pages/blog/release-logs.md]: Applied

File: src/pages/blog/release-logs.md (hunk 2 above)
Comment (bryanoltman) [reply src/pages/blog/release-logs.md]: Applied

**Review by eseidel — APPROVED** [review]
Looks great!

File: src/pages/blog/viewing-logs.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+and
+[Windows](https://stackoverflow.com/questions/20564514/adb-is-not-recognized-as-an-internal-or-external-command-operable-program-or)
+
+Without any filtering, `adb logcat` is _very_ noisy. To show just the output for
+your Flutter app, you can filter the output using `adb logcat | grep flutter` on
+macOS/Linux or `adb logcat | findstr flutter` on Windows.
+
+![](../../assets/images/blog/release-logs/TerminalLogcat.png)
+
+## Console.app for iOS
```
[the remark this answers is not in the document]
Comment (bryanoltman) [reply src/pages/blog/viewing-logs.md:39]: Mac desktop is basically the same as iPhone and would be an easy addition. Interested to see what Windows desktop would look like (for this and a million other reasons)

**Pull request merged**